### PR TITLE
Feature nxp 23314 ordered folder

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

File: test/nuxeo-move-documents-buttons.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  <script>
+    suite('Given on a Ordered Folder, I have 5 items', function() {
+      var server;
+      var downButton, upButton;
+      var items = [
+        {"uid": "1"},
+        {"uid": "2"},
+        {"uid": "3"},
+        {"uid": "4"},
+        {"uid": "5"}
```
Comment (Gabez0r) [inline test/nuxeo-move-documents-buttons.html]: use single instead of double quotes

File: actions/nuxeo-move-documents-up-button.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        this._available = false;
+        if (this.selectedDocuments && this.selectedDocuments.length > 0) {
+          // Let's sort
+          this._sortedDocuments = this.selectedDocuments.slice(0);
+          try {
+            this._sortedDocuments.sort(function(a, b) {
+              var idxa = this.documents.indexOf(a);
+              var idxb = this.documents.indexOf(b);
+              if (idxa < 0 || idxb < 0) {
+                throw new Error("Document is not in the list.");
```
Comment (Gabez0r) [inline actions/nuxeo-move-documents-up-button.html]: use single instead of double quotes

File: actions/nuxeo-move-documents-down-button.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        this._available = false;
+        if (this.selectedDocuments && this.selectedDocuments.length > 0) {
+          // Let's sort
+          this._sortedDocuments = this.selectedDocuments.slice(0);
+          try {
+            this._sortedDocuments.sort(function(a, b) {
+              var idxa = this.documents.indexOf(a);
+              var idxb = this.documents.indexOf(b);
+              if (idxa < 0 || idxb < 0) {
+                throw new Error("Document is not in the list.");
```
Comment (Gabez0r) [inline actions/nuxeo-move-documents-down-button.html]: use single instead of double quotes

File: actions/nuxeo-move-documents-down-button.html [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+      iron-icon:hover {
+        fill: var(--nuxeo-link-hover-color);
+      }
+    </style>
+
+    <nuxeo-operation id="moveDownOp"
+      op="Document.Order"
+      params='{"before": "[[_beforeUid]]"}'
+      headers='{"nx_es_sync": "true"}'
```
Comment (Gabez0r) [inline actions/nuxeo-move-documents-down-button.html]: leverage new `sync-indexing` property

File: actions/nuxeo-move-documents-up-button.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+      iron-icon:hover {
+        fill: var(--nuxeo-link-hover-color);
+      }
+    </style>
+
+    <nuxeo-operation id="moveUpOp"
+      op="Document.Order"
+      params='{"before": "[[_beforeUid]]"}'
+      headers='{"nx_es_sync": "true"}'
```
Comment (Gabez0r) [inline actions/nuxeo-move-documents-up-button.html]: leverage new `sync-indexing` property

**Comment by Gabez0r** [comment]
👍

**Comment by afjusto** [comment]
👍

**Pull request merged**